index.bs

```diff
@@ -532,9 +532,9 @@ Add the step:
 
 # Fenced Frame monkeypatches # {#fenced-frame-monkeypatches}
 
-Add the following items to [=fencedframetype/pending event=]:
+Add the following items to [=fencedframetype/destination enum event=]:
 
-<dl dfn-for="pending event">
+<dl dfn-for="destination enum event">
```

Comment:
Hmm, why are these two lines necessary to fix the build? A "pending event" still exists in the FF spec: https://wicg.github.io/fenced-frame/#fencedframetype-pending-event.

Reply:
Yeah, but I think "destination enum event" is where these two properties are needed.

Reply:
OK, so this isn't related to the "fixing a broken build", this is just a general spec change that's included here?

Reply:
Thanks Dom. The build error is due to accessing the eventType/eventData field of pending event, but I refactored with the latest fenced frame spec.